### ISSUE TITLE
Refuse to load Google Tag Manager

### ISSUE THREAD

**Issue by MichalTuleja** [LiskHQ/lisk-explorer#828]
### Expected behavior
In order to collect the usage data the Google Tag Manager scripts have to be loaded properly.

### Actual behavior
Current security policy refuses to load GTM scripts (see the screenshot).

### Steps to reproduce
Load the page

![screenshot 2018-11-19 at 09 49 03](https://user-images.githubusercontent.com/4547510/48695789-5f5e0900-ebe0-11e8-86f6-ecdac61db1c4.png)

**Comment by MichalTuleja** [comment]
Another problem:
```
6721.js?428510:1 Refused to load the script 'https://s3.amazonaws.com/trk.cetrk.com/b/t.js?t=1542638913171' because it violates the following Content Security Policy directive: "script-src 'self' 'unsafe-eval' 'unsafe-inline' https://tagmanager.google.com/ https://www.googletagmanager.com/ https://www.google-analytics.com/ http://*.cloudfront.net/ https://*.ipify.org/ https://*.crazyegg.com".
```

**Comment by MichalTuleja** [comment]
Solved in `v2.1.8`.
